src/Core/Routes.js

```diff
@@ -32,6 +32,7 @@ const RegistrationContainer = lazy(() => import('../Modules/Family/RegistrationC
 const AgencyEventListContainer = lazy(() => import('../Modules/Events/AgencyEventListContainer'));
 const EventDetailsContainer = lazy(() => import('../Modules/Events/EventDetailsContainer'));
 const RegistrationConfirmComponent = lazy(() => import('../Modules/Family/RegistrationConfirmComponent'));
+const PantryContainer = lazy(() => import('../Modules/Pantry/PantryContainer'));
```

Comment:
as discussed since its new route for Home Page. 
can you please update this to Home Module/Container

Reply:
Yes Added in another branch